crates/autopilot/src/domain/settlement/trade/math.rs

```diff
@@ -479,26 +444,12 @@ impl Trade {
     }
 }
 
-#[derive(Debug, Clone)]
-pub struct Prices {
-    pub uniform: ClearingPrices,
-    /// Adjusted uniform prices to account for fees (gas cost and protocol fees)
-    pub custom: ClearingPrices,
-}
-
```

Comment:
moved to `domain::settlement::transaction::mod.rs`

Reply:
any reason? I personally think clearing prices make more sense in the context of a trade than in the context of a transaction?

Reply:
Because now we have `EncodedTrade` type which is a plain object, very public, so it makes sense to put all simple types that are used there as well, while math::Trade is supposed to be private/closed

Reply:
I wouldn't have put it in `math::Trade` but in the highlevel thing that hosts the enum. But also no strong feeling